docs/3.0-migration-guide.d/ccm-alt.md

```diff
@@ -0,0 +1,8 @@
+CCM interface changes: impact for alternative implementations
+-------------------------------------------------------------
+
+The CCM multipart interface has changed with the addition of support for
```

Comment:
I think this "multipart" should be removed.

Reply:
done